### PR TITLE
IDEMPIERE-3561 QuickEntry panel doesn't use ZoomCondition

### PR DESCRIPTION
**Pull request by nmicoud**
I've added migration scripts in ID10 folder.
Not sure if they should have been in i9

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java [hunk 1]
```diff
@@ -2225,7 +2225,24 @@ protected int getADWindowID() {
...
+				if (isSOTrx.equals("Y") && userDef.getAD_Window_ID() > 0)
+					return userDef.getAD_Window_ID();
+				else if (isSOTrx.equals("N") && userDef.getPO_Window_ID() > 0)
+					return userDef.getPO_Window_ID();	
+			}
+
+			if (isSOTrx.equals("Y") && infoWindow.getAD_Window_ID() > 0)
+				return infoWindow.getAD_Window_ID();
+			else if (isSOTrx.equals("N") && infoWindow.getPO_Window_ID() > 0)
+				return infoWindow.getPO_Window_ID();
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java:2243]: for isSOTrx.equals("N"), we should return infoWindow.getAD_Window_ID() if infoWindow.getPO_Window_ID()== 0 and infoWindow.getAD_Window_ID() > 0

**Review by hengsin — CHANGES_REQUESTED** [review]

**Comment by hengsin** [comment]
Hi @CarlosRuiz-globalqss, do you wants to add this to 9 ?

**Comment by CarlosRuiz-globalqss** [comment]
> Hi @CarlosRuiz-globalqss, do you wants to add this to 9 ?

Yes, I think is a bug fix, sounds better for 9

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java (hunk 1 above)
Comment (nmicoud) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java]: done

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**